lib/h3core.js

```diff
@@ -469,6 +488,22 @@ export function h3GetBaseCell(h3Index) {
     return H3.h3GetBaseCell(lower, upper);
 }
 
+/**
+ * Get the indices of all icosahedron faces intersected by a given H3 index
+ * @static
+ * @param  {H3Index} h3Index  H3 index to get faces for
+ * @return {Number[]}         Indices (0-19) of all intersected faces
+ */
+export function h3GetFaces(h3Index) {
+    const [lower, upper] = h3IndexToSplitLong(h3Index);
+    const count = H3.maxFaceCount(lower, upper);
+    const faces = C._malloc(SZ_INT * count);
+    H3.h3GetFaces(lower, upper, faces);
```

Comment:
Did we ever add an error return type to the C code? I recall Isaac bringing that up. If so, this should have a `const err = ...` in front and an appropriate `throw` if non-zero.

Reply:
We didn't add an error code, because I didn't think it made sense to do input validation here - the caller may choose to filter all input by `h3IsValid`, but if this step is unnecessary it seems like they ought to be able to skip it to avoid the perf cost. Most of our functions taking an H3 index don't validate input, though I definitely think we should revisit this in the next major version of the core lib.